code/modules/surgery/organs/heart.dm

```diff
@@ -220,21 +220,12 @@ obj/item/organ/heart/cybernetic/upgraded/on_life()
 	addtimer(VARSET_CALLBACK(src, dose_available, TRUE), 5 MINUTES)
 	ramount = 0
 
-
-
 /obj/item/organ/heart/ipc
 	name = "IPC heart"
 	desc = "An electronic pump that regulates hydraulic functions, they have an auto-restart after EMPs."
 	icon_state = "heart-c"
 	organ_flags = ORGAN_SYNTHETIC
 
-/obj/item/organ/heart/ipc/emp_act()
```

Comment:
This was discussed. It allows the IPC hearts to restart when ion'd. Why are you removing this?

Reply:
Because this was not mentioned in the PR title, body, or any comments.